modules/installation-performing-disconnected-mirror.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc
+
+[id="installation-performing-disconnected-mirror"]
+= Performing a mirror to disk for use in disconnected environments
+
+When you mirror images to disk, you download images as files. Then, you move your
+mirror host, which can be a laptop or a removable media device, like a
+USB drive, into the restricted network and complete the mirror
+procedure.
+
+.Procedure
+
+. On a local host, test the mirror process:
++
+----
+$ oc adm release mirror <product_version> --to <mirror_repository> --dry-run
```

Comment:
From my understanding, "Mirroring the OpenShift Container Platform image repository" is mirror-to-registry scenario, "Mirroring the content" is mirror-to-disk scenario. They are should be at the same level. So can we re-arrange the doc structure to make the two scenarios as some kind of list for easy reading? Such as:
Mirroring the OpenShift Container Platform image repository:

- mirror release image to local registry directly

- mirror release image to disk files, then upload files to local registry

BTW, is it possible to make 'product_version' and 'mirror_repository' be more specific, just like what we did in "Mirroring the OpenShift Container Platform image repository" section to introduce environment variables, so that make mirror-to-registry and mirror-to-disk in a consistent style.

Reply:
@jianlinliu, if I make the introduction cover both use cases, can we just use this module instead? https://github.com/openshift/openshift-docs/pull/21993/files#diff-ac4ad58a22744e32fc45f760386799bbR1

Reply:
https://github.com/openshift/openshift-docs/pull/21993/files#diff-ac4ad58a22744e32fc45f760386799bbR1 focus on upgrade, but not install. To some content, whatever upgrade or fresh install, the mirror commands may be the same. But there is sill some difference in the whole process. For example, install need a detailed steps about how to record imageContentSources, and inject it into install-config.yaml;  install section should never have "--apply-release-image-signature" option when mirror image to local registry, which is only applicable for upgrade scenario.

I went through the preview doc, I am okay with its layout, especially in "3. Mirror the version images to the internal container registry". But maybe need more polish for new "mirror-to-disk" scenario with detailed steps, especially we should try our best to avoid new issues for existing "mirror-to-reigstry" scenario (so far its steps is working perfectly).

Reply:
@jianlinliu, will you please take another look? I had a chat with Trevor about trying to get the imageContentSources information for the "mirror-to-disk" scenario, and there's not an automatic way to do it. He said that the "mirror-to-disk" commands that are in the reworked module (and the refreshed preview build) should work. I took the "mirror-to-registry" commands from before and reformatted it all to follow the same format as the upgrade steps.